src/types.d.ts

```diff
@@ -44,7 +44,9 @@ export interface ISignExternalTxRequest {
 }
 
 export interface ISigner {
-  send(to: string, amount: number, options: ISendTxOptions): Promise<Insight.ISendRawTxResult>;
+  send(to: string, amount: number, transactionSpeed: string): Promise<Insight.ISendRawTxResult>;
```

Comment:
`options: ISendTxOptions` should be what you need to use. you have to pass in an object with `{ feeRate: 500 }`.

if there are already defined types, lets use those.

Reply:
`ISigner` was meant to be a generic interface that we can use later for other signers (like a hardware wallet) so we need to use the rawest arguments possible. `TransactionSpeed` is something that is only applicable for Qrypto.